app/controlplane/api/controlplane/v1/workflow.proto

```diff
@@ -101,7 +101,16 @@ message WorkflowServiceDeleteRequest {
 
 message WorkflowServiceDeleteResponse {}
 
-message WorkflowServiceListRequest {}
+message WorkflowServiceListRequest {
+  optional string project_name = 1 [(buf.validate.field) = {
```

Comment:
How about supporting smth like project_ref that can be either the id or the name? 

The reason is to make this API suitable for clients that use IDs instead of names?

Reply:
Sure thing will do!